### DISCUSSION TITLE
Example for fetching data and serversided pagination

### DISCUSSION THREAD

**Discussion by Code-Case** [quasarframework/quasar#9320] [General - Components / Directives / etc] [answered]
Hello,

Im trying to understand where should I place the Axios api call in the following codepen, currently I have it this way but then the pagination isnt working right, or are am I at the wrong position to get my data and do something with it?

The example isnt really clear for me where I should fetch the data

Example: https://codepen.io/metalsadman/pen/xxxJKmJ

My try with api fetching https://codepen.io/someOne11/pen/bGqwQwY

it works but Im not sure if this is the right way or place :)

best and thank you

**Comment by metalsadman** [comment, marked as answer]
i don't see no wrong in it, other than implementing the sorting to the serverside.

**Reply by Code-Case** [reply]
Thanks for checking.
